static/styles/explorer.scss

```diff
@@ -548,14 +548,23 @@ input.vim-check {
 }
 
 /* While we find a better solution, lets up this a bit */
-.compiler-picker .ts-dropdown-content {
-    max-height: 300px !important;
+.compiler-picker-dropdown .ts-dropdown-content {
+    max-height: 600px !important;
```

Comment:
Having a taller dropdown is really nice but tomselect doesn't do anything to prevent it going off the screen... I'll want to investigate this more before merging.

Reply:
Handled :)